test/suites/integration/tlsroute_serviceexport_test.go

```diff
@@ -0,0 +1,150 @@
+package integration
+
+import (
+	"fmt"
+	"log"
+	"net"
+	"os"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/vpclattice"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/samber/lo"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+
+	gwv1 "sigs.k8s.io/gateway-api/apis/v1"
+	"sigs.k8s.io/gateway-api/apis/v1alpha2"
+	"sigs.k8s.io/gateway-api/apis/v1beta1"
+
+	"github.com/aws/aws-application-networking-k8s/pkg/model/core"
+	"github.com/aws/aws-application-networking-k8s/test/pkg/test"
+
+	anv1alpha1 "github.com/aws/aws-application-networking-k8s/pkg/apis/applicationnetworking/v1alpha1"
+)
+
+var _ = Describe("TLSRoute ServiceExport ServiceImport Test", func() {
+	var (
+		deployment1   *appsv1.Deployment
+		service1      *v1.Service
+		tlsRoute      *v1alpha2.TLSRoute
+		serviceExport *anv1alpha1.ServiceExport
+		serviceImport *anv1alpha1.ServiceImport
+		policy        *anv1alpha1.TargetGroupPolicy
+	)
+
+	It("create a tlsRoute2serviceimport", func() {
+		deployment1, service1 = testFramework.NewHttpsApp(test.HTTPsAppOptions{Name: "my-https-1", Namespace: k8snamespace})
+		policy = createTCPTargetGroupPolicy(service1)
+		testFramework.ExpectCreated(ctx, policy)
+		serviceImport = testFramework.CreateServiceImport(service1)
+		testFramework.ExpectCreated(ctx, serviceImport)
+		serviceExport = testFramework.CreateServiceExport(service1)
+		testFramework.ExpectCreated(ctx, serviceExport)
+
+		tlsRoute = testFramework.NewTLSRoute(k8snamespace, testGateway, []v1alpha2.TLSRouteRule{
+			{
+				BackendRefs: []gwv1.BackendRef{
+					{
+						BackendObjectReference: v1beta1.BackendObjectReference{
+							Name:      v1alpha2.ObjectName(service1.Name),
+							Namespace: lo.ToPtr(v1beta1.Namespace(service1.Namespace)),
+							Kind:      lo.ToPtr(v1beta1.Kind("ServiceImport")),
+							Port:      lo.ToPtr(v1beta1.PortNumber(443)),
+						},
+					},
+				},
+			},
+		})
+
+		// Create Kubernetes API Objects
+		testFramework.ExpectCreated(ctx,
+			tlsRoute,
+			service1,
+			deployment1,
+		)
+		route, _ := core.NewRoute(tlsRoute)
+		vpcLatticeService := testFramework.GetVpcLatticeService(ctx, route)
+		fmt.Printf("vpcLatticeService: %v \n", vpcLatticeService)
+
+		tgSummary := testFramework.GetTCPTargetGroup(ctx, service1)
+		tg, err := testFramework.LatticeClient.GetTargetGroup(&vpclattice.GetTargetGroupInput{
+			TargetGroupIdentifier: aws.String(*tgSummary.Id),
+		})
+		Expect(tg).To(Not(BeNil()))
+		Expect(err).To(BeNil())
+		Expect(*tgSummary.VpcIdentifier).To(Equal(os.Getenv("CLUSTER_VPC_ID")))
+		Expect(*tgSummary.Protocol).To(Equal("TCP"))
+		if tg.Config.HealthCheck != nil {
+			Expect(*tg.Config.HealthCheck.Enabled).To(BeFalse())
```

Comment:
Since we already need a TG policy here, would it be a good idea to include an enabled health check to make sure that's being set up properly?